test/fixtures/testProvenanceUtils.js

```diff
@@ -0,0 +1,469 @@
+const DELEGATOR_WHO = {
+  reference: 'Practitioner/456',
+  type: 'Practitioner'
+};
+const EXPECTED_DELEGATOR = {
+  type: {
+    coding: [
+      {
+        system: 'http://terminology.hl7.org/CodeSystem/v3-RoleCode',
+        code: 'DELEGATOR',
+        display: 'delegator'
+      }
+    ]
+  },
+  role: {
+    coding: [
+      {
+        system: 'http://terminology.hl7.org/CodeSystem/v3-RoleCode',
+        code: 'DELEGATOR',
+        display: 'delegator'
+      }
+    ]
+  },
+  who: {
+    reference: 'Practitioner/456',
+    type: 'Practitioner'
+  },
+  requestor: true
+};
+
+const SINGLE_AGENT_PROVENANCE = {
+  resourceType: 'Provenance',
+  occurredPeriod: {
+    start: '2015-01-01',
+    end: '2015-01-02'
+  },
+  recorded: '2015-02-07T13:28:17.239+02:00',
+  policy: 'https://www.policy.com',
+  reason: [
+    {
+      coding: [
+        {
+          system: 'http://terminology.hl7.org/CodeSystem/v3-ActReason',
+          code: 'CAREMGT',
+          display: 'care management'
```

Comment:
```suggestion
          code: 'HQUALIMP',
          display: 'health quality improvement'
```

The code we want here is the [health quality improvement one](https://www.hl7.org/fhir/v3/ActReason/cs.html#v3-ActReason-HQUALIMP), not care management. It looks like there's no hardcoding of this so that's good, you'll just have to update your test fixtures. I'll make suggestions to make your life easier and you can batch them